wagtail/admin/rich_text/editors/draftail/__init__.py

```diff
@@ -29,6 +35,7 @@ def __init__(self, *args, **kwargs):
             plugin = feature_registry.get_editor_plugin('draftail', feature)
             if plugin:
                 plugin.construct_options(self.options)
+                self._media += plugin.media
```

Comment:
Does this guarantee that `draftail.js` will be loaded before the plugin's JS?

Reply:
I think so, yes. It definitely does in Django 2.0: https://docs.djangoproject.com/en/2.0/topics/forms/media/#order-of-assets

Hopefully the note about earlier versions ("concatenated rather than merged") just means that 1.11 can't handle fancy cyclic dependencies like the one in the example, rather than not preserving order at all - but I'll double-check that.

Reply:
Yep, on Django 1.11 the `+` operation appends items in the original order: https://github.com/django/django/blob/stable/1.11.x/django/forms/widgets.py#L104

Reply:
Sweet :) thanks for tracking it down